src/sentry/runner/commands/upgrade.py

```diff
@@ -1,12 +1,45 @@
 import click
 from django.conf import settings
+from django.db import connections
 
 from sentry.runner.decorators import configuration
 
 
+def _check_history():
+    connection = connections["default"]
+    cursor = connection.cursor()
+    try:
+        # If the database has 0 tables we are good to go.
+        cursor.execute(
+            """
+            SELECT COUNT(*) AS total FROM information_schema.tables
+            WHERE table_catalog = %s AND table_schema = 'public'
+            """,
+            [settings.DATABASES["default"]["NAME"]],
+        )
+        if cursor.fetchone()[0] == 0:
+            return
+    except Exception:
+        raise click.ClickException("Could not determine migration state. Aborting")
```

Comment:
What do you think about also printing the original exception message to help with debugging?